source/docs/01-introduction.xml

```diff
@@ -277,6 +281,31 @@
                      available and used for all customizations officially provided by MEI.</p>
+                     <p>With <hi rend="bold">MEI 5.1</hi>, five additional elements have been added to the 
+                        schema: <gi scheme="MEI">tuning</gi>, <gi scheme="MEI">course</gi>, 
+                        <gi scheme="MEI">string</gi>, <gi scheme="MEI">tabDurSym</gi>, and 
+                        <gi scheme="MEI">tabGrp</gi>. All of these elements are used to encode tablature
+                     notation. The use of these elements, including relevant attributes, is documented in the
+                     largely rewritten chapter <ptr target="#tablature"/> of these Guidelines. The existing
+                        <ident type="class">att.stringtab.tuning</ident> attribute class has been deprecated
+                        in favor of these additions and will be removed from a future version of MEI.</p>
+                     <p>The new <ident type="class">att.dataSelecting</ident> class (currently available 
+                        only on <gi scheme="MEI">symbolDef</gi> introduces an XPath-based selection 
+                        mechanism to identify elements to which certain features apply. All controlevents
+                        (see <ptr target="#eventsControlevents"/>, technically implemented through membership in the
+                        <ident type="class">att.controlEvent</ident> class) are now member of 
+                        <ident type="class">att.color</ident>, making it possible to use the <att>color</att>
+                        attribute on them. Similarly, directive-like elements (<gi scheme="MEI">bend</gi>, 
+                        <gi scheme="MEI">breath</gi>, <gi scheme="MEI">caesura</gi>, <gi scheme="MEI">cpMark</gi>, 
+                        <gi scheme="MEI">fermata</gi>, <gi scheme="MEI">fing</gi>, <gi scheme="MEI">fingGrp</gi>, 
+                        <gi scheme="MEI">harm</gi>, <gi scheme="MEI">harpPedal</gi>, <gi scheme="MEI">metaMark</gi>, 
+                        <gi scheme="MEI">mordent</gi>, <gi scheme="MEI">octave</gi>, <gi scheme="MEI">ornam</gi>,
+                        <gi scheme="MEI">trill</gi>, and <gi scheme="MEI">turn</gi>) are now member of the
+                        <ident type="class">att.verticalGrp</ident> class, which gives them access to the <att>vgrp</att>
+                        attribute, which is used to align these elements vertically.</p>
+                     <p>Minor changes include the addition of <att>visible</att> an <att>enclose</att> on 
+                        <gi scheme="MEI">meterSigGrp</gi>. To see a full comparison of changes from MEI 5.0 to MEI 5.1.,
+                        please consider <ref target="https://github.com/music-encoding/music-encoding/compare/v5.0...v5.1">https://github.com/music-encoding/music-encoding/compare/v5.0...v5.1</ref></p>
```

Comment:
```suggestion
                        <gi scheme="MEI">meterSigGrp</gi>.</p>
```

Reply:
We don't have such a specific reference to v4.0.1...v5.0, so wouldn't introduce it for v5.1 here.

Reply:
that was available in your MarkDown, and seemed pretty useful ;-)